packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -307,10 +307,34 @@ const Redirect = observer(() => {
     }
     useEffect(() => {
         if (!redirected_to_route && history.location.pathname !== routes.traders_hub) {
-            history.push({
-                pathname: routes.traders_hub,
-                search: url_query_string,
-            });
+            if (/dtrader/i.test(history.location.search)) {
+                history.push({
+                    pathname: routes.trade,
+                    search: url_query_string,
+                });
+            } else if (/smart_trader/i.test(url_query_string)) {
```

Comment:
For Smart Trader, we need to put this logic in SmartTrader repo?

Reply:
I will make changes to smart_trader repo accordingly